src/pages/en/installation-new.md

```diff
@@ -27,16 +30,28 @@ pnpm create astro
 > ⚔️ Prefer to go it alone? Read our [manual setup](/en/guides/manual-setup-new) instructions instead.
 
 
-If `create-astro` starts successfully, you will see a list of [starter templates](https://github.com/withastro/astro/tree/main/examples) to choose from: 
+If `create-astro` starts successfully, you will see a short list of starter templates to choose from: 
 - `starter`: A great starter template for anyone wanting to explore Astro.
 - `minimal`: A template that just includes the bare minimium to get started.
 - `blog, portfolio, docs, etc`: opinionated themes for specific use-cases.
 
 If you choose the `starter` template, you will also be asked to select which [additional frameworks](/en/core-concepts/component-hydration) (React, Svelte, Vue, Solid, Preact), if any, you would like to include in your project. (Additional frameworks can also be added manually later.)
 
+> 💡 Or, you can install any of our [many starter templates](https://github.com/withastro/astro/tree/main/examples) directly via the command line: 
```

Comment:
large quotes like this don't look great in our current stylesheet. Okay to move out of a quote?

Reply:
Done.